src/tiled/scriptfileinfo.cpp

```diff
@@ -242,3 +242,74 @@ QString ScriptFileInfo::symLinkTarget(const QString &file) const
 {
     return QFileInfo(file).symLinkTarget();
 }
+
+QString ScriptFileInfo::cleanPath(const QString &file) const
+{
+    return QDir().cleanPath(file);
+}
+
+QString ScriptFileInfo::fromNativeSeparators(const QString &file) const
+{
+    return QDir().fromNativeSeparators(file);
+}
+
+QString ScriptFileInfo::toNativeSeparators(const QString &file) const
+{
+    return QDir().fromNativeSeparators(file);
+}
+
+bool  ScriptFileInfo::cd(const QString &file)
+{
+    return QDir().cd(file);
+}
+
+uint  ScriptFileInfo::count(const QString &file) const
+{
+    return QDir(file).count();
+}
+
+QString ScriptFileInfo::dirName(const QString &file) const
+{
+    return QDir(file).dirName();
```

Comment:
This is really the same as `fileName`, so I'd suggest we leave it out.

Reply:
sounds good.